responsibleai/responsibleai/_tools/data_balance/__init__.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
```

Comment:
Could you may be defined the BaseDataBalanceService class in separate python file? Seems weird that we are defining a python class in __init__.py.